repository/src/test/java/org/alfresco/repo/content/ContentServiceImplUnitTest.java

```diff
@@ -78,6 +79,8 @@
     private static final String X_AMZ_HEADER_2 = "x-amz-header-2";
     private static final String VALUE_2 = "value2";
 
+    private static final QName QNAME = ContentModel.PROP_CONTENT;
```

Comment:
Trivial: it's not just any QNAME - maybe rename to PROP_CONTENT_QNAME to be explict.